sklearn/datasets/descr/boston_house_prices.rst

```diff
@@ -50,4 +50,4 @@ problems.
 
    - Belsley, Kuh & Welsch, 'Regression diagnostics: Identifying Influential Data and Sources of Collinearity', Wiley, 1980. 244-261.
    - Quinlan,R. (1993). Combining Instance-Based and Model-Based Learning. In Proceedings on the Tenth International Conference of Machine Learning, 236-243, University of Massachusetts, Amherst. Morgan Kaufmann.
-   - many more! (see http://archive.ics.uci.edu/ml/datasets/Housing)
+   - many more! (see https://archive.ics.uci.edu/ml/machine-learning-databases/housing/)
```

Comment:
remove directly? I can't find any more there

Reply:
Yeah its the only one there.
Should I remove line 53 completely?

Reply:
I think so

Reply:
Removed!